### PR TITLE
add 2 filter for blockqoute and sentence code

### PR DESCRIPTION
**Pull request by aureliokessler**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,6 +45,18 @@ const filters = {
...
+  blockqoteCodeFilter: {
+    regex: /```.*```/s,
+    run: (message, tag) => {
+      return message.content.replace(/```.*```/s, 'code');
+    }
+  },
+  CodeFilter: {
+    regex: /`\w+.*`/g,
+    run: (message, tag) => {
+      return message.content.replace(/`\w+.*`/g, 'code');
```
Comment (lucaelin) [inline index.js]: ``"`foo`bar`".replace(/`\w+.*`/, 'code')`` results in `"code"` but it should result in ``"codebar`"``

File: index.js [hunk 2]
```diff
@@ -45,6 +45,18 @@ const filters = {
     run: (message, tag) => {
       return message.content.replace(/https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/, 'url');
     }
+  },
+  blockqoteCodeFilter: {
+    regex: /```.*```/s,
+    run: (message, tag) => {
+      return message.content.replace(/```.*```/s, 'code');
```
Comment (lucaelin) [inline index.js]: ````"```foo```bar```".replace(/```.*```/, 'code')```` results in `"codesdad"` but should result in ````"codebar```"````

**Review by lucaelin — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (aureliokessler) [reply index.js]: that fix it ``"`foo`bar`".replace(/`\w+.*?`/, 'code')`` results ``"codebar"``

second test case: ``"`foo`bar`foo`bar`".replace(/`\w+.*?`?/, 'code')`` returns ``"codebar`foo`bar`"`` this not fixed.

File: index.js [hunk 3]
```diff
@@ -47,15 +47,15 @@ const filters = {
     }
   },
   blockqoteCodeFilter: {
-    regex: /```.*```/s,
+    regex: /(`{3}[a-z]*\n[\s\S]*?\n`{3})/,
```
Comment (lucaelin) [inline index.js]: ``/`{3}((?!`{3}).)+`{3}/`` seems to be the only working solution...

**Review by lucaelin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**